src/pages/workspaces/workspace/Dashboard/Dashboard.ts

```diff
@@ -103,7 +102,7 @@ interface WorkspaceDashboardComponentProps extends WorkspaceDashboardProps {
   workspace: Workspace;
 }
 
-const WorkspaceDashboardComponent = (
+const WorkspaceDashboardForwardRefRenderFunction = (
```

Comment:
This isn't a component, it's a ForwardRefRenderFunction. React's forwardRef takes a ForwardRefRenderFunction and returns a component.

Reply:
nit: would ForwardRefRenderer be a slightly less lenghthy postfix on this?  Also, since it's not a component, shouldn't it start with lower-case 'w'?

Reply:
ForwardRefRenderFunction was taken from the type. Also, generally I prefer "noun" over "verb doer" for names.

But yeah, since it's not a component, maybe `renderWorkspaceDashboard` would be preferable.

Reply:
Never mind, ESLint gets unhappy about calling hooks from a function whose name doesn't start with either a capital letter or "use".

Reply:
Alternatively, this could be wrapped in `forwardRef` to remove the need for a name for the render function. But that creates a much larger diff, so I'm not inclined to do that here.